### PR TITLE
Delete support for old controlplane in policy engine

### PR DESCRIPTION
**Pull request by zivnevo**

### CODE REVIEW

**Comment by orozery** [comment]
@kfirtoledo Have we decided it's time to remove the old controlplane/dataplane?

**Comment by kfirtoledo** [comment]
Yes, that's what @elevran  said in the last sync meeting.

File: pkg/policyengine/PolicyDispatcher.go [hunk 1]
```diff
@@ -50,7 +47,6 @@ const (
 )
```
Comment (kfirtoledo) [inline pkg/policyengine/PolicyDispatcher.go:47]: Can you remove also MBG acronyms from the code.

**Review by kfirtoledo — CHANGES_REQUESTED** [review]
You still need the accessControl.go or can you remove it

File: pkg/policyengine/PolicyDispatcher.go (hunk 1 above)
Comment (zivnevo) [reply pkg/policyengine/PolicyDispatcher.go:44]: Done

**Comment by zivnevo** [comment]
> You still need the accessControl.go or can you remove it

Removed it. Good catch.

**Review by kfirtoledo — APPROVED** [review]

**Pull request merged**